openedx_events/event_bus/__init__.py

```diff
@@ -183,3 +184,28 @@ def make_single_consumer(*, topic: str, group_id: str,
 def _reset_state(sender, **kwargs):  # pylint: disable=unused-argument
     """Reset caches when settings change during unit tests."""
     get_producer.cache_clear()
+
+
+def merge_producer_configs(producer_config_original, producer_config_overrides):
```

Comment:
NIT: renaming these variables to `original` and `overrides` would make it easier to understand as `producer_config` prefix should not be needed.